inotify/adapters.py

```diff
@@ -46,6 +47,16 @@ def __init__(self, paths=[], block_duration=_DEFAULT_EPOLL_BLOCK_DURATION_S):
         for path in paths:
             self.add_watch(path)
 
+    @staticmethod
+    def to_bytes(s):
+        text = unicode if sys.version_info.major == 2 else str
+        return s.encode(sys.getdefaultencoding()) if isinstance(s, text) else s
+
+    @staticmethod
+    def to_unicode(s):
+        bytestr = str if sys.version_info.major == 2 else bytes
```

Comment:
`bytes` is equivalent to `str` in Python 2:

```
Python 2.7.9 (default, Apr 16 2015, 14:01:56) 
[GCC 4.2.1 Compatible Apple LLVM 6.1.0 (clang-602.0.49)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> bytes is str
True
```

So, I would use `bytes`, if defined, or, if not defined (*NameError*), use `str`.